packages/synapse-react-client/src/components/UserCard/UserBadge.tsx

```diff
@@ -51,20 +56,26 @@ export const UserCardSmall = (props: UserCardSmallProps) => {
     SynapseConstants.USER_BUNDLE_MASK_IS_CERTIFIED |
     SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED
 
+  const { data: userGroupHeader } = useGetUserGroupHeader(userId!, {
+    enabled: Boolean(userId),
+  })
```

Comment:
Uses the `UserGroupHeader`, so the data for this component can now be pre-fetched in a batch!